init.lua

```diff
@@ -187,17 +183,21 @@ for _, v in pairs({
 		color = "#7dea1eff",
 		tiles = {tile},
 		place_param2 = 136,
+        visual_scale = 2.3,
```

Comment:
where did this come from?

Reply:
It allow grass to overlap each other a little when seen at ground level. There is wider grass textures  to preview the concept.  Currently 64px early sketch. But If you like I'll provide final one likely to be 32px to match the visual style of the default game.